tsconfig.json

```
@@ -4,7 +4,7 @@
     "checkJs": true,
     "forceConsistentCasingInFileNames": true,
     "noImplicitReturns": true,
-    "noImplicitAny": true,
+    "noImplicitAny": false,
```

@Gozala how strongly do we  feel about this?
@jay-dee7 is it possible to add type info and keep this as `true`?
```suggestion
    "noImplicitAny": true,
```

---

I just added type information for the methods where typescript transpiler was failing and reverted this option back to `true`. It works now as expected :)